doc/source/User_guide/mesh.rst

```diff
@@ -20,3 +20,40 @@ are listed in the mesh object:
   :width: 800
   :alt: Mesh object List
 
+
+Icepak has a different approach to the mesh operations.
+Those are managed through mesh regions and they can be edited directly from the pyaedt object.
```

Comment:
```suggestion
Those are managed through mesh regions and can be edited directly from the pyaedt object.
```
If ``pyaedt`` is a code entity, place in double backticks. Otherwise, change to "PyAEDT".